File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    # 문자열을 공백을 기준으로 나누어 숫자 배열로 저장
    l = list(map(int, s.split(" ")))
    # 숫자 배열에서 최솟값과 최댓값을 구한 뒤 숫자로 문자열로 저장 
    return str(min(l)) + " " + str(max(l))
    

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
import java.util.*;
class Solution {
    public String solution(String s) {
        /*
        " "으로 스플릿해서 string 만든 뒤에 Integer.parseInt해서 정수배열로 바꾸기
        stream 이용해서 max, min 쓰기?
        */
        String[] sArray = s.split(" ");
        Integer[] intArray = new Integer[sArray.length];
        for(int i=0; i<sArray.length;i++){
            intArray[i] = Integer.parseInt(sArray[i]);
            System.out.println(intArray[i]);
        }
        int max = Arrays.stream(intArray).max(Integer::compare).get();
        int min = Arrays.stream(intArray).min(Integer::compare).get();
        
        
        return Integer.toString(min) + " " + Integer.toString(max);
    }
}